lake-primitives/src/types/transactions.rs

```diff
@@ -12,12 +12,12 @@ pub struct Transaction {
     receiver_id: AccountId,
     status: ExecutionStatus,
     execution_outcome_id: CryptoHash,
-    operations: Vec<Operation>,
+    // actions: Vec<ActionKind>,
```

Comment:
Here's the most interesting part. Please have a look at all the actions, at the `ActionKind` enum, at the `Action` trait first.

So having all that I just cannot reuse that structures here anymore, since they have denormalized `metadata` containing the data from a `Receipt`. Here for `Transaction` I don't have a `Receipt` yet.

@frol believes it is not even needed here, so we should omit the field `actions` from the `Transaction` structure completely. I am convinced, although I would like to know if there are any second thoughts.

Reply:
@khorolets Wait, what is this struct about? I was suggesting exposing `Transaction` type as is, with the near_pritimitive ActionView inside. I did not suggest leaving users completely without access to the transaction actions.

Reply:
Let's discuss it synchronously then.